### ISSUE TITLE
the dim of F.softmax

### ISSUE THREAD

**Issue by sicheng7** [utkarsh2254/modelling-adversarial-perturbations#2]
The dimension of the “prob_vec_no_shuffle” is (16,1000), why is the dim of the F.softmax function set to 0 ？
